### PR TITLE
Remove TCP entry point

### LINKED ISSUE #987: Deprecate TCP in favor of WebSocket
Currently we have the choice between TCP and WebSocket. WebSocket provide a simpler method of dealing with network damage and is the preferred method already to use for WebAO. It also reduce potential confusion for the less knowledgeable users

### PR DESCRIPTION
**Pull request by TrickyLeifa**
Resolve #987

* Remove TCP entry point
  * Servers that do not support WebSocket will be marked as `Legacy`
  * Removal of TCP connection from the master will follow later.

### CODE REVIEW

**Comment by TrickyLeifa** [comment]
deprecation-tcp
As mentioned, example of the marking.

**Comment by Salanto** [comment]
Maybe we should reword this tooltip more for the end user? Most people will have no clue what a websocket entry point is.
Something like "Unable to connect to server due to missing WebSocket support"?

**Comment by TrickyLeifa** [comment]
Sure.

**Review by Salanto — APPROVED** [review]
Hee-Hoo pilled.
